apps/checkout/public/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Saleor Checkout Frontend SPA</title>
   </head>
-  <body>
+  <body spellcheck="false">
```

Comment:
This will disable spell checking for inputs that can inherit the `spellcheck` attribute. Do we actually want to do that, i.e. disable spell checking for all fields by default?

Reply:
Yes, why not?

Reply:
oh, ok, I thought we wanted that only for specific fields